src/compressor/Compressor.h

```diff
@@ -16,7 +16,10 @@
 #define CEPH_COMPRESSOR_H
 
 #include "include/int_types.h"
-#include "include/Context.h"
+#include "include/memory.h"
+#include "include/buffer.h"
```

Comment:
`Compressor.h` is included by other files also. by using `buffer_fwd.h` speed up the compilation, i don't see it has anything to do with `SnappyCompressor.h`. maybe i am missing anything?